tool/grind.dart

```diff
@@ -462,11 +433,11 @@ Future<void> compareSdkWarnings() async {
       Directory.systemTemp.createTempSync('dartdoc-comparison-sdkdocs');
   var originalDartdoc = createComparisonDartdoc();
   var currentDartdocSdkBuild = _buildSdkDocs(
-      sdkDocsDir.path, Future.value(Directory.current.path), 'current');
+      _sdkDocsDir.path, Future.value(Directory.current.path), 'current');
   var originalDartdocSdkBuild =
       _buildSdkDocs(originalDartdocSdkDocs.path, originalDartdoc, 'original');
   var currentDartdocWarnings = jsonMessageIterableToWarnings(
-      await currentDartdocSdkBuild, sdkDocsDir.absolute.path, null, 'HEAD');
+      await currentDartdocSdkBuild, _sdkDocsDir.absolute.path, null, 'HEAD');
```

Comment:
It would be nice to make all these getters return absolute objects. Even if we sometimes print relative, it should be an explicit operation - print something relative.